app/helpers/media_objects_helper.rb

```diff
@@ -223,7 +223,7 @@ def parse_node section, node, tracknumber
          native_url = "#{pid_section_media_object_path(@mediaobject, section.pid)}?t=#{start},#{stop}"
          url = "#{share_link_for( section )}?t=#{start},#{stop}"
          data =  {segment: section.pid, is_video: section.is_video?, native_url: native_url, fragmentbegin: start, fragmentend: stop}
-         link = link_to label, url, data: data, class: 'playable wrap'+(' current-stream' if is_current_section?(section))
+         link = link_to label, url, data: data, class: 'playable wrap'+(is_current_section?(section) ? ' current-stream' : '' )
```

Comment:
I'm curious was this change purely aesthetic or did it actually change something?